js/app.js

```diff
@@ -25,13 +29,32 @@
    * on this dom element, set the className to the value of desiredClass
    */
 
+  function setMyLightClass(event, desiredClass) {
+    let lightId = this.dataset.lightId;
+    document.getElementById(lightId).className = desiredClass;
+  }
 
   /*
    * Declare 15 constants btn1, btn2, btn3, ...
    * use a dom element selector method to assign each const
    * to the button that has an id of the same name.
    */
 
+  const btn1 = document.querySelector("#btn1");
```

Comment:
Although this works, there is a more specific document method that allows you to get an element by id.